pandas/tests/test_dtypes.py

```diff
@@ -80,6 +80,13 @@ def test_basic(self):
         self.assertFalse(is_categorical(np.dtype('float64')))
         self.assertFalse(is_categorical(1.0))
 
+    def test_series_with_dtype(self):
+        self.assertRaises(
+            ValueError, lambda: Series(Categorical([1, 2, 3]), dtype='int64'))
+        s = Series(Categorical([1, 2, 3]), dtype='category')
+        self.assertTrue(is_categorical_dtype(s))
```

Comment:
You mean moving the contents of this test into test_constructor_categorical() in tests/series/test_constructor?